### PR TITLE
SETTING UP

### PR DESCRIPTION
**Pull request by gracebir**
### This PR is about setting up and cleaning up the project

Here we are going to install the necessary packages and define some tools that 
we are going to use within this project.

here are the steps:

- [x] Installing necessary packages
- [x] define colors 
- [x] create constants file
- [x] Create an animation file
- [x] Create a landing page skeleton

### CODE REVIEW

File: src/app/_components/About.tsx
Comment (merhmood) [inline src/app/_components/About.tsx:1]: why are using _components naming convention

File: package.json
Comment (merhmood) [inline package.json:1]: 🔥

File: src/app/layout.tsx
Comment (merhmood) [inline src/app/layout.tsx:1]: The metadata description for the src/app/layout.tsx still has the create Next app description.

File: src/constants/index.ts
Comment (merhmood) [inline src/constants/index.ts:1]: i was think about creating some kind of static json file that will be fetched into the pages that require those data.

**Review by merhmood — COMMENTED** [review]
Thanks for the cleaning up

**Comment by merhmood** [comment]
The commit messages are mostly chores

File: src/app/_components/About.tsx
Comment (gracebir) [reply src/app/_components/About.tsx:1]: @merhmood, in Next.js 13 start the folder with an `underscore` in the `app` folder means that the folder will not be going to considered like a `page directory.`

**Pull request merged**